src/secureSignalEuid.ts

```diff
@@ -0,0 +1,19 @@
+import { isDebugModeOn, UidSecureSignalProvider } from './secureSignal_shared';
+
+const INTEG_BASE_URL = 'https://cdn.integ.uidapi.com/';
```

Comment:
This should use the EUID cdn - this script won't exist there yet. https://cdn.integ.euid.eu/

Reply:
changed